.travis.yml

```diff
@@ -0,0 +1,13 @@
+sudo: False
+language: python
+python:
+  - "3.5"
+  - "3.4"
+  - "2.7"
+  - "2.6"
```

Comment:
I'm okay with removing it. I like to support 2.6 on my projects since it's usually not much hassle, but  it's becoming less relevant.